randovania/games/am2r/pickup_database/item_data.json

```diff
@@ -0,0 +1,176 @@
+{
+	"Missile Expansion": {
+		"sprite_name": "sItemMissile",
+		"sprite_speed": 0.2,
+		"text_desc": "Missile capacity {MissilesChanged} by {Missiles}",
+		"text_header": "Got Missile Tank"
```

Comment:
all of this should be in the extra dicts in the real pickup database instead of a separate file. also, `sprite_name` is redundant when the pickup definition's `model_name` exists

Reply:
How should I handle this for offworld items? I.e ETM, dread ice missiles etc.? My current thought is modifying `_get_item_data` to do the following:
- get all relevant data from extra fields in DB
- add the duplicates for things that are in the db (i.e. expansion, locked stuff and so on)
- manually construct offworld items via some dict

but not sure whether that's the best approach.

Reply:
ah, you're right, a separate lookup dict is still appropriate here

Reply:
So keep it as is then (with the exception of the `sprite_name` field), or how am I to understand it?

Reply:
:+1:

Reply:
Should be done for everything in the am2r folder, a PR doing it for the db's outside the am2r folder will follow soon:tm: